change/@fluentui-react-toolbar-694f03cb-529d-4cf1-8923-11ac8f126074.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "minor",
```

The same for `dependentChangeType`.